cpp/tests/utilities/column_utilities.cu

```diff
@@ -427,6 +427,23 @@ struct column_view_printer {
     this->template operator()<cudf::string_view>(*col_as_strings, out, indent);
   }
 
+  template <typename Element, typename std::enable_if_t<cudf::is_fixed_point<Element>()>* = nullptr>
+  void operator()(cudf::column_view const& col,
+                  std::vector<std::string>& out,
+                  std::string const& indent)
+  {
+    auto const h_data = cudf::test::to_host<Element>(col);
+
+    out.resize(col.size());
+    std::transform(thrust::make_counting_iterator(size_type{0}),
+                   thrust::make_counting_iterator(col.size()),
+                   out.begin(),
+                   [&](auto idx) {
+                     auto const d = static_cast<double>(h_data.first[idx]);
```

Comment:
Should we add an `operator<<` overload for the fixed_point type? Casting to `double` will cause rounding which means it won't print the exact value. Seems like we should have an `operator<<` for the fixed-point type that takes care to print the _exact_ value.

Reply:
There actually was a `operator<<` overload before this (but it merely did the same thing). @devavret mentioned that have `#include <iostream>` might have been an issue - so I just removed it. I can look into adding (a more sophisticated one back).

Reply:
Have set up a follow up issue for this so as not to block https://github.com/rapidsai/cudf/issues/5747